model/host/host.go

```diff
@@ -150,6 +150,7 @@ type Host struct {
 	SSHKeyNames []string `bson:"ssh_key_names,omitempty" json:"ssh_key_names,omitempty"`
 
 	IsVirtualWorkstation bool `bson:"is_virtual_workstation" json:"is_virtual_workstation"`
+	IsCluster            bool `bson:"is_cluster" json:"is_cluster"`
```

Comment:
I don't know that we need this value. because the distro is included in the host document on line 35, it might make things simpler to not have this field duplicated.

Reply:
Agreed. I thought it was likely I might make some superfluous change like this. I will fix